ui-kit/src/main/res/values/typography.xml

```diff
@@ -1,120 +1,118 @@
         <item name="android:textSize">11sp</item>
         <item name="android:letterSpacing">0.01</item>
     </style>
 
-    <style name="TextAppearance.Regular.Label1" parent="TextAppearance.Regular">
+    <style name="UiKit.TextAppearance.Regular.Label1" parent="UitKit_TextAppearance.Regular">
         <item name="android:textSize">12sp</item>
     </style>
 
-    <style name="TextAppearance.Regular.Caps" parent="TextAppearance.Regular">
+    <style name="UiKit.TextAppearance.Regular.Caps" parent="UitKit_TextAppearance.Regular">
         <item name="android:textSize">12sp</item>
         <item name="android:textAllCaps">true</item>
     </style>
 
-    <style name="TextAppearance.Regular.Text4" parent="TextAppearance.Regular">
+    <style name="UiKit.TextAppearance.Regular.Text4" parent="UitKit_TextAppearance.Regular">
         <item name="android:textSize">13sp</item>
     </style>
 
-    <style name="TextAppearance.Regular.Text3" parent="TextAppearance.Regular">
+    <style name="UiKit.TextAppearance.Regular.Text3" parent="UitKit_TextAppearance.Regular">
         <item name="android:textSize">15sp</item>
     </style>
 
-    <style name="TextAppearance.Regular.Text2" parent="TextAppearance.Regular">
+    <style name="UiKit.TextAppearance.Regular.Text2" parent="UitKit_TextAppearance.Regular">
         <item name="android:textSize">16sp</item>
     </style>
 
-    <style name="TextAppearance.Regular.Text1" parent="TextAppearance.Regular">
+    <style name="UiKit.TextAppearance.Regular.Text1" parent="UitKit_TextAppearance.Regular">
         <item name="android:textSize">17sp</item>
     </style>
 
-    <style name="TextAppearance.Regular.Title3" parent="TextAppearance.Regular">
+    <style name="UiKit.TextAppearance.Regular.Title3" parent="UitKit_TextAppearance.Regular">
         <item name="android:textSize">20sp</item>
         <item name="android:letterSpacing">0.02</item>
     </style>
 
-    <style name="TextAppearance.Regular.Title2" parent="TextAppearance.Regular">
+    <style name="UiKit.TextAppearance.Regular.Title2" parent="UitKit_TextAppearance.Regular">
         <item name="android:textSize">22sp</item>
         <item name="android:letterSpacing">0.02</item>
     </style>
 
-    <style name="TextAppearance.Regular.Title1" parent="TextAppearance.Regular">
+    <style name="UiKit.TextAppearance.Regular.Title1" parent="UitKit_TextAppearance.Regular">
         <item name="android:textSize">28sp</item>
         <item name="android:letterSpacing">0.01</item>
     </style>
 
-    <style name="TextAppearance.Regular.LargeTitle" parent="TextAppearance.Regular">
+    <style name="UiKit.TextAppearance.Regular.LargeTitle" parent="UitKit_TextAppearance.Regular">
         <item name="android:textSize">34sp</item>
         <item name="android:letterSpacing">0.01</item>
     </style>
     <!--endregion-->
 
     <!--region TextAppearance.SemiBold-->
-    <style name="TextAppearance.SemiBold">
+    <style name="UitKit_TextAppearance.SemiBold">
```

Comment:
why UitKit_TextAppearance and then UiKit.TextAppearance ?

Reply:
you mean why UitKit_TextAppearance and not UiKit.TextAppearance?

Reply:
yes! =)

Reply:
check again please, is it ok for you?